variables.tf

```diff
@@ -113,9 +82,24 @@ variable "topics" {
   description = "List of Hedwig topics"
   default     = {}
   type = map(object({
-    # Firehose all messages published to this topic into GCS
+    # Firehose—that is, save—all messages published to this topic into GCS
     enable_firehose = optional(bool)
 
+    # Variable firehose_bucket declares the bucket for firehose—that is, saved—messages (should already exist)
+    firehose_bucket = optional(string)
```

Comment:
Same suggestion/q. from the other PR : https://github.com/cloudchacho/terraform-google-hedwig-topic/pull/2#discussion_r1417627468

Reply:
See https://github.com/cloudchacho/terraform-google-hedwig/pull/4/commits/8b8ff6899c5faf8fb1d3acc9d1d8fed04d2d282f commit.